### PR TITLE
DBZ-8162 Fix open redo log consistency with reopening nodes

### PR DESCRIPTION
**Pull request by Naros**
https://issues.redhat.com/browse/DBZ-8162

### CODE REVIEW

**Comment by jpechane** [comment]
Test failures unrelated.
@Naros Applied, thanks!

File: debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/logminer/LogFileCollectorTest.java [hunk 1]
```diff
@@ -1810,6 +1811,104 @@ public void testOpenRedoThreadSwitchDoesNotMissArchiveLogIfNotYetAvailable() thr
...
+                .thread()
+                .threadId(2)
+                .status("OPEN")
+                .enabled("PUBLIC")
+                .enabledScn(Scn.valueOf(2028329))
+                .checkpointScn(Scn.valueOf(14338976116789L))
+                .lastRedoScn(Scn.valueOf(14339038344484L))
+                .lastRedoSequenceNumber(138697L)
+                .build()
+                .build();
```
Comment (spicy-sauce) [inline debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/logminer/LogFileCollectorTest.java:1839]: why twice?

Comment (jpechane) [reply debezium-connector-oracle/src/test/java/io/debezium/connector/oracle/logminer/LogFileCollectorTest.java:1839]: The first call builds `RedoThread` and the second builds final `RedoThreadState`.

**Pull request merged**